extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -316,6 +324,46 @@ export function initializeWizardPage(context: WizardPageContext): void {
 	});
 }
 
+/**
+ * Hooks up the dynamic enablement for fields which use that. This will attach a listener to the target component
+ * for when the value changes and update the enabled state of the source component based on the current value
+ * of the target component.
+ *
+ * Note that currently this is only supported for Notebook Wizard Pages and only supports direct equals comparison
+ * for the value currently selected.
+ *
+ * Additionally this only supports hooking up components that are on the same page.
+ * @param context The page context
+ */
+function hookUpDynamicEnablement(context: WizardPageContext): void {
+	context.pageInfo.sections.forEach(section => {
```

Comment:
In vscode that variable was showing as 'any' for me, adding an explicit type would be useful

Reply:
What variable? Because this is all typed correctly for me. Are you sure that vs code had finished parsing the project? It takes a while after you switch branches to do that.

Reply:
Yeah it is possible that vscode was acting funny when i reviewed so ok to ignore this.